### PR TITLE
Prevent glog output on errors

### PR DESCRIPTION
**Pull request by fgimenez**
Closes https://github.com/giantswarm/giantswarm/issues/3266

Basically follow https://github.com/kubernetes/kubernetes/issues/17162#issuecomment-225596212 to prevent log entries like:
```
ERROR: logging before flag.Parse: E0525 13:00:15.500040       1 portforward.go:331] an error occurred forwarding 42217 -> 44134: error forwarding port 44134 to pod cc9240723af2a191e17e5149cde3b6436a2fe80d107e09e3e22a0e63e7581a22, uid : exit status 1: 2018/05/25 13:00:15 socat[3420] E connect(5, AF=2 127.0.0.1:44134, 16): Connection refused
```

Unit test still missing.

### CODE REVIEW

**Comment by fgimenez** [comment]
Tested locally and on gauss.

File: Gopkg.toml [hunk 1]
```diff
@@ -34,7 +34,9 @@
   name = "k8s.io/client-go"
   version = "kubernetes-1.10.4"
 
-
+[[constraint]]
+  name = "k8s.io/apimachinery"
+  version = "kubernetes-1.10.4"
```
Comment (indyfree) [inline Gopkg.toml]: Just wondering why do we need `k8s.io/apimachinery` here if we just add a `flag....` command?

Comment (fgimenez) [reply Gopkg.toml]: You are totally right :) this comes from the initial approaches. Removed.

**Review by indyfree — APPROVED** [review]
Little bit of a hack, but LGTM ;)

**Pull request merged**